app/controllers/coronavirus_form/basic_care_needs_controller.rb

```diff
@@ -27,6 +27,6 @@ def submit
 private
 
   def previous_path
-    carry_supplies_path
+    @form_responses[:essential_supplies] == I18n.t("coronavirus_form.questions.essential_supplies.options.option_no.label") ? carry_supplies_path : essential_supplies_path
```

Comment:
This line is really long. Perhaps it could be split up? 
e.g.

```
return carry_supplies_path if @form_responses[:essential_supplies] == I18n.t("coronavirus_form.questions.essential_supplies.options.option_no.label")

essential_supplies_path
```